src/test/java/Homework17.java

```diff
@@ -10,22 +10,47 @@
 public class Homework17 extends LoginTests {
 
     @Test
-    public static void addSongToPlaylist() {
+    public void addSongToPlaylist() {
+
+        LoginValidEmailValidPasswordTest();
+        //clickCreatePlaylist();
+        //inputPlaylist();
+        //enterPress();
+        //openHomeURL();
+        clickViewAll();
+        clickFirstSong();
+        clickAddto();
+        clickPlaylistfromAddto();
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        wait.until(ExpectedConditions.visibilityOf(getSuccessPopup()));
+        Assert.assertEquals(getSuccessNotif(),"Added 1 song into test.");
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221399768-deb625a3-edf2-47f7-83a9-d0b2e3318837.png)

**Hint**  please enter your email and password

![image](https://user-images.githubusercontent.com/104761482/221399814-44f5cca7-e2b5-4139-833f-1b5128128dca.png)